internal/nodes/avi_model_nodes.go

```diff
@@ -909,12 +909,9 @@ type AviVSVIPNode struct {
 	Tenant                  string
 	CloudConfigCksum        uint32
 	FQDNs                   []string
-	EastWest                bool
 	VrfContext              string
 	IPAddress               string
-	SubnetIP                string
-	SubnetPrefix            int32
-	NetworkNames            []string
+	VipNetworks             []akov1alpha1.AviInfraSettingVipNetwork
```

Comment:
Directly referring a CRD field here may not be required. We can create a struct with two strings and use it. 
Just a suggestion, functionality wise it is fine